lib/stack_master/identity.rb

```diff
@@ -1,16 +1,21 @@
 module StackMaster
   class Identity
-    def running_in_allowed_account?(allowed_accounts)
-      allowed_accounts.nil? || allowed_accounts.empty? || allowed_accounts.include?(account)
+    def running_in_account?(accounts)
+      accounts.nil? ||
+        accounts.empty? ||
+        contains_account_id?(accounts) ||
+        contains_account_alias?(accounts)
     end
 
     def account
       @account ||= sts.get_caller_identity.account
     end
 
-    private
+    def account_aliases
+      @aliases ||= iam.list_account_aliases.account_aliases
```

Comment:
With the addition of this new API call, I'm assuming StackMaster requires additional permissions. Perhaps we should include this in the changelog.

Reply:
Added in 1fc44b7.